app/views/crops/search.html.haml

```diff
@@ -1,6 +1,6 @@
 - content_for :title, "Crops matching #{@search}"
 
-- unless (@exact_match || @partial_matches.length > 0)
+- unless (@all_matches.length > 0)
```

Comment:
Maybe we can refactor this into a simpler conditional. `if @all_matches` ... `else`